frontend/src/components/chats/components/current-chat/components/message-form/helpers/check-message-has-only-white-spaces.helper.ts

```diff
@@ -0,0 +1,7 @@
+const checkMessageHasOnlyWhiteSpaces = (message: string): boolean => {
+  const onlyWhiteSpacesPattern = /^\s*$/;
+
+  return onlyWhiteSpacesPattern.test(message);
```

Comment:
Can't it just be trimmed and casted to Boolean?

Reply:
agreed